requirements.txt

```diff
@@ -12,7 +12,7 @@ expandvars==0.9.0
 fastapi[all]==0.89.1
 fastapi-caching[redis]==0.3.0
 fastapi-pagination[sqlalchemy]~= 0.10.0
-fideslang==1.4.1
+fideslang @ git+https://github.com/ethyca/fideslang.git@6e93ab4ea7d35713201d48b3314b8d25df0815cb
```

Comment:
Temporary commit until we release fideslang again - 

Separately, there are some test failures on this branch here that seem unrelated to my particular fideslang changes. 

@ThomasLaPiana in particular, this field:  `security_policy: https://ethyca.com/privacy-policy/` seems to be problematic now in `demo_resources/demo_organization.yml`.  I am not sure what this means.

Missing resource keys: {HttpUrl('https://ethyca.com/privacy-policy/', scheme='https', host='ethyca.com', tld='com', host_type='domain', port='443', path='/privacy-policy/')}

Potentially introduced here: https://github.com/ethyca/fideslang/commit/526c3fdbc8736051d89a28965061ddaa757807d9

Reply:
as part of this update, we should probably unpin `pydantic` completely or pin it aggressively, I have a feeling that's what caused this

Reply:
although it shouldn't be thinking that field is a FidesKey, which is what the reference check would look for...

We _did_ recently update that logic though, I can open a bug ticket and check

Reply:
ah thanks for weighing in Thomas! that would be great if you could open a bug ticket for this

Reply:
bug ticket here: https://github.com/ethyca/fideslang/issues/130